src/node/ui_interface.h

```diff
@@ -82,6 +82,9 @@ class CClientUIInterface
     /** Progress message during initialization. */
     ADD_SIGNALS_DECL_WRAPPER(InitMessage, void, const std::string& message);
 
+    /** Wallet client created. */
+    ADD_SIGNALS_DECL_WRAPPER(InitWallet, void, );
```

Comment:
any reason for the trailing comma?

Reply:
> any reason for the trailing comma?

This is to fix the clang warning `./node/ui_interface.h:86:46: error: must specify at least one argument for '...' parameter of variadic macro [-Werror,-Wgnu-zero-variadic-macro-arguments]
    ADD_SIGNALS_DECL_WRAPPER(InitWallet, void);` https://cirrus-ci.com/task/6303923532201984